### PR TITLE
Add Longhorn 1.2.6 chart with dev-2.6

### PR DESCRIPTION
**Pull request by weizhe0422**
Signed-off-by: Ray Chang <ray.chang@suse.com>

self test
 - Installation v1.2.6 
    - [x] Fresh install
    - [x] Fresh install and upgrade to v1.3.2
    - [x] Upgrade from v1.2.5
 -  Function
     - [x] Deploy statefulset (replica count = 2)
     - [x] Scale down to 1 => one volume detached; scale up to 3 => three volumes attached 
     - [x] Write data in the pod
     - [x] Take snapshot
     - [x] Take backup and restored
  
  cc @innobead @PhanLe1010

### CODE REVIEW

**Comment by innobead** [comment]
cc @khushboo-rancher

File: charts/longhorn/100.1.4+up1.2.6/values.yaml [hunk 1]
```diff
@@ -0,0 +1,237 @@
...
+        taintToleration: cattle.io/os=linux:NoSchedule
+        systemManagedComponentsNodeSelector: kubernetes.io/os:linux
+
+image:
+  longhorn:
+    engine:
+      repository: longhornio/longhorn-engine
+      tag: v1.2.6
+    manager:
+      repository: longhornio/longhorn-manager
```
Comment (khushboo-rancher) [inline charts/longhorn/100.1.4+up1.2.6/values.yaml:30]: These should be mirrored images

**Review by khushboo-rancher — CHANGES_REQUESTED** [review]

File: charts/longhorn/100.1.4+up1.2.6/values.yaml (hunk 1 above)
Comment (weizhe0422) [reply charts/longhorn/100.1.4+up1.2.6/values.yaml]: I didn't add `value.yaml.patch` to replace it with mirror images, I have fixed it.

Please help to review it again. Thank you.

Comment (PhanLe1010) [reply charts/longhorn/100.1.4+up1.2.6/values.yaml]: Hi Ray. this is still not fixed yet. The output here should contain mirrored images values

**Comment by PhanLe1010** [comment]
There are some patch files that are removed in packages/longhorn/longhorn-1.2/generated-changes/patch

It seems that did not make run `make patch` after `make prepare`

**Comment by weizhe0422** [comment]
Close this PR since duplicate with https://github.com/rancher/charts/pull/2197.

**Pull request closed without merging**